hibernate-core/src/main/java/org/hibernate/engine/spi/SessionImplementor.java

```diff
@@ -36,7 +36,7 @@
  *         {@link org.hibernate.resource.jdbc.spi.JdbcSessionContext} delegate
  *     </li>
  *     <li>
- *         {@link TransactionCoordinatorBuilder.Options}
+ *         {@code Options} interface in {@link TransactionCoordinatorBuilder}
```

Comment:
There is no possibility to link to inner interface within parent interface in current Javadoc, thus the change.